compiler/prelude/prelude.go

```diff
@@ -1,10 +1,10 @@
+// Prelude is the GopherJS JavaScript interop layer that is behind
+// the github.com/gopherjs/gopherjs/js package
+const Prelude = prelude + numeric + types + goroutines + jsmapping
 
```

Comment:
I think this time is ok (I think @shurcooL has already confirmed), but I'd separate this PR into moving this part and generating this part so that we could confirm the generated code was same.

Reply:
Oh, https://github.com/gopherjs/gopherjs/pull/791 is the moving part? I missed that, sorry.

Reply:
Yes #791 is separate. Unless we want to collapse this and #791 into one PR. Which makes sense from my perspective.